Sources/AppcuesKit/Presentation/Extensions/UIViewController+Embed.swift

```diff
@@ -19,7 +19,7 @@ extension UIViewController {
     func unembedChildViewController(_ childVC: UIViewController) {
         guard childVC.parent == self else { return }
         childVC.willMove(toParent: nil)
-        childVC.removeFromParent()
         childVC.view.removeFromSuperview()
+        childVC.removeFromParent()
```

Comment:
To get `.isMovingFromParent` to work correctly on disappear, the `removeFromParent()` needs to be called after the `view.RemoveFromSuperview()`.  Not breaking anything currently, I don't think, but would make life easier in the future to have organized this way so that we can properly track dismissal/close analytics on any non-modal content.

Reply:
I'm not sure why we had it in that order since Apple's code examples match what you've changed it to, so no concerns about this breaking anything: https://developer.apple.com/library/archive/featuredarticles/ViewControllerPGforiPhoneOS/ImplementingaContainerViewController.html